website/docs/reference/global-configs/databricks-changes.md

```diff
@@ -8,19 +8,108 @@ The following are the current [behavior change flags](/docs/reference/global-con
+We plan to switch the default of this flag to `True` in v1.11.0.
+
+## Use restructured materializations
+
+The `use_materialization_v2` flag is `False` by default and guards significant rewrites of the core materializations in `dbt-databricks` while they are still in an experimental stage.
+
+We plan to switch the default of this flag to `True` in 1.11.0, depending on user feedback.
+
+### Changes to the Seed materialization
+
+The seeds materialization should have the smallest difference between the old and new materialization, as the primary difference is just removing calls to methods that are not supported by Databricks, such as transaction operations.
+
+### Changes to the View materialization
+
+In conjunction with the flag above, there are two model configuration options that can customize how we handle the view materialization when we detect an existing relation at the target location.
+
+#### `view_update_via_alter`
+
+For view models where we set `view_update_via_alter: True`, when we detect an existing view, rather than using `create or replace...` to replace the existing view, we gather metadata to determine if the existing view matches the dbt project.
+When it does, no further action is taken; otherwise, `alter view...` is used to alter the view in place to match your dbt project.
+This allows continuity of history for the view, which is lost with `create or replace...`, which can fix user-reported issues with Unity Catalog features.
+
+:::caution There is currently no support for altering the comment on a view via Databricks SQL.
+
+As such, we must replace the view whenever you change its description
+
+:::
+
+#### `use_safer_relation_operations`
+
+The other new model configuration introduced for views is `use_safer_relation_operations`.
+When this configuration is set to `True`, if `view_update_via_alter` is not set, we will use a multi-step, renaming approach to replacing existing relations.
+Rather than using a `create or replace...` statement to replace an existing view, we create the new view at a staging location, then swap locations with the existing view, before finally deleting the existing view.
+
+:::caution This configuration option may increase storage costs.
+
+While this approach is equivalent to the default dbt view materialization, it will create additional UC objects, as compared to alternatives.
+
+:::
+
+While this setting is more relevant for table and incremental materializations, there are benefits for use with views.
+For example, when replacing a materialized view with a traditional view, we have historically dropped the materialized view prior to creating the new view.
+Although materialized views do not currently support renaming, setting this config ensures that the materialized view is not dropped until we have successfully created the new view in the staging location; as such, there is no longer a scenario where the existing relation is dropped and the new relation fails to be created.
+
```

Comment:
i think for both model configs, it'd be great to have an example of the behavior flag enabled, as well as the model configs enabled so users know how this config looks?